etc/oracle-admin/procedures.sql

```diff
@@ -0,0 +1,34 @@
+/*
+Some procedures that dario found on production DB, which may be useful in the future
```

Comment:
what about the montlhy partition dropping ? Aren't those stored procedures as well ?

Reply:
OK. those are in the "partitons.sql"